bpf/lib/process.h

```diff
@@ -357,6 +357,17 @@ validate_msg_execve_size(int64_t size)
 	return size;
 }
 
+// execve_map_error() will increment the map error counter
+static inline __attribute__((always_inline)) void execve_map_error(void)
+{
+	int one = 1;
```

Comment:
Maybe this is too pedantic but could we use a named constant here like MAP_STATS_ERROR for the error key and similarly MAP_STATS_COUNT for the size key?

I think it makes the purpose of the two keys more clear for those who want to use or extend it in the future.

Reply:
sure that makes sense to me.